### PR TITLE
LPS-143163 PoC to add React Fragment in site-initializer

### PR DESCRIPTION
**Pull request by jairmedeiros**

### CODE REVIEW

**Comment by kevenleone** [comment]
It's look real good, @jairmedeiros 

Is this working?

Dreaming with that for Raylife 🥰🥰🥰

**Comment by claricebrasil** [comment]
Just starting reviewing :)

**Comment by claricebrasil** [comment]
Just starting reviewing :)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/liferay-npm-bundler.config.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @liferay/imports-first */
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/liferay-npm-bundler.config.js:1]: Remove this line

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "extra-liferay-fragments",
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/package.json:2]: Probably, this is a bad name.

Following the existing pattern, would be:

`@liferay/osb-site-initializer-customer-portal-fragments`

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/src/select-project/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    });
+  }, []);
+
+  return (
+    <ClayDropDownWithItems
+      footerContent={<ClayButton>All Projects</ClayButton>}
+      items={filteredItems}
+      onSearchValueChange={setValue}
+      searchValue={value}
+      searchable={true}
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/src/select-project/index.js:39]:
```suggestion
      searchable
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/src/select-project/index.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+import ClayButton from '@clayui/button';
+import { ClayDropDownWithItems } from '@clayui/drop-down';
+import React, { useEffect, useState } from 'react';
+
+const spritemap = Liferay.ThemeDisplay.getCDNBaseURL() + "/o/admin-theme/images/clay/icons.svg";
+
+export default function () {
+  const [items, setItems] = useState([]);
+  const [value, setValue] = useState('');
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/src/select-project/index.js:20]: What value means??

Maybe a different name

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/src/select-project/index.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+  const filteredItems = items.filter((item) => item.label.toLowerCase().includes(value.toLowerCase()));
+
+  useEffect(() => {
+    window.addEventListener('customer-portal-select-user-loading', ({ detail: {accountBriefs} }) => {
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/src/select-project/index.js:25]: You need to remove the eventListener, inside the useEffect

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Pull request closed without merging**